R/doeFactorial.R

```diff
@@ -110,9 +110,9 @@ doeFactorial <- function(jaspResults, dataset, options, ...){
                        factors = options[["numberOfFactors"]],
                        runs = runs,
                        resolution = resolution,
-                       centers = options[["factorialCenterPoints"]],
-                       replicates = options[["factorialCornerReplicates"]],
-                       blocks = options[["factorialBlocks"]])
+                       centers = "NA",
+                       replicates = "NA",
+                       blocks = "NA")
```

Comment:
as long as the default values don't crash, you only need to hide it in the QML file.

Reply:
the default values are inaccurate with some of the input options, so I've hidden them for now